### ISSUE TITLE
Command & modal exit

### ISSUE THREAD

**Issue by vincentcordel** [livewire/flux#508]
When using command and modal, it's possible to exit the command modal using escape. 
It should be also possible to exit by clicking anywhere outside of the modal.

**Comment by calebporzio** [comment]
You're right, this is a broader issue: Browser's `<dialog>` element does not allow this behavior - there are proposals for it, but nothing in browsers yet.

Therefore, I think we should build this behavior ourselves for all modals in Flux.

I'm going to leave this open until that is done. Thanks.

**Comment by joalea** [comment]
Hey @calebporzio,

Sorry for being late on this when you have already implemented the change, but I would argue that the closing when clicking outside the modal should be optional. I have plenty of scenarios where I don't want the modal to be closed when someone miss clicks.

Anyways, great work om Flux! Love it <3

**Comment by calebporzio** [comment]
@joalea - good call. the option will be in the next release:

![Image](https://github.com/user-attachments/assets/fd882850-3180-4808-bad8-cf30cf232947)
